AU2/plugins/util/DeathManager.py

```diff
@@ -3,7 +3,7 @@
 from AU2.database.model import Event, Assassin
 
 
-class DeathManager:
+class DeathManager:  # TODO Undo changes; reimplemented inside wantedmanager

```

Comment:
undo what changes?

Reply:
Good catch, I modified it for the first wanted rewrite, but death manager is no longer used, so those changes can be reverted